manifests/0000_50_cluster_monitoring_operator_04-config.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+data:
+  metrics.yaml: |-
+    "matches":
+    - "{__name__=\"up\"}"
```

Comment:
So the procedure to add a new item to the allowed list, would be to just modify this ConfigMap?

Reply:
correct

Reply:
Do you think we can document this either in the google doc we have or in the README, can be done outside of this PR, just so we don't have to explain to people. :)

Reply:
nit: we should also document the expectations about changes of that configmap in a running cluster, i.e. what happens if somebody (or some system) tries to modify that configmap.

Reply:
not a nit, I think this is required, will fix

Reply:
and once this is merged I will point the current doc at the one in this repo

Reply:
Did you update this list to reflect the current state in telemeter repo?

Reply:
Currently this is still being generated. As soon as this lands we would remove this being generated, but until then I'd like to keep this to ensure that when this lands this is definitely in sync with what is currently merged in telemeter.

tl;dr yes